[mega branch] main spring launch event PR

mirnawong1 opened the pull request:
This PR and branch serves as the main (mega) branch the docs team will use to merge their changes into for the spring launch showcase event. 

Some of the changes inlcude:
- lifecycle status updates
- product-related updates
- spring launch joint release notes

Commit:
remove preview mentions

Commit:
lifecycle chnages/updates

Commit:
add release note for spring launch event

Commit:
remove 'new'

nghi-ly committed:
Add info for Fabric and Azure

nghi-ly committed:
PM feedback on Azure

nghi-ly committed:
Reworded Fabric to match

Commit:
dbt assist addtions

Commit:
updates

Commit:
updates

mirnawong1 committed:
Update website/docs/docs/build/unit-tests.md

Commit:
update

Commit:
update header

Commit:
remove space

Commit:
greg's updates

Commit:
update

Commit:
feedback

Commit:
update

Commit:
remove s

mirnawong1 committed:
Merge branch 'current' into spring-launch-dbt-assist

Commit:
add info

Commit:
add info about repo

Commit:
update link

Commit:
update to links

mirnawong1 committed:
Merge branch 'current' into spring-launch-dbt-assist

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-power-user

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-dbt-assist

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-ide-filesystem

Commit:
remove caching

matthewshaver committed:
Merge branch 'current' into spring-launch-release-notes

matthewshaver committed:
Fixing release note location

Commit:
remove

matthewshaver committed:
Updating based on meeting

matthewshaver committed:
Merge branch 'spring-launch-release-notes' of https://github.com/dbt-labs/docs.getdbt.com into spring-launch-release-notes

nghi-ly committed:
Add info for Explorer

nghi-ly committed:
Updated wording for Azure a tiny bit

Commit:
azzam feedback

Commit:
update to ga

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-release-notes

Commit:
update after meeting

Commit:
update

Commit:
update rn

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-lifecycle-updates

matthewshaver committed:
Apply suggestions from code review

runleonarun committed:
Merge branch 'current' into spring-launch-24

runleonarun committed:
Merge branch 'spring-launch-24' into spring-launch-dbt-assist

matthewshaver committed:
Merge branch 'spring-launch-24' into spring-launch-power-user

mirnawong1 committed:
Update website/docs/docs/cloud/about-cloud/about-dbt-cloud.md

matthewshaver committed:
add link to dbt power user (#5479)

this pr adds a link to dbt power user for the cloud cli. [docs
project](https://www.notion.so/dbtlabs/DevExp-Enhancements-CLI-IDE-Git-AI-9d72cc68bcc44a6bafe95eab793235fb?pvs=4)

matthewshaver committed:
Update website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md

matthewshaver committed:
Update website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md

matthewshaver committed:
Update develop-in-the-cloud.md

Spacing is causing formatting problems

matthewshaver committed:
ide info about file size repo limit (#5480)

this pr adds info / consideration for repo file sizes over 10gb.

[notion
project](https://www.notion.so/dbtlabs/DevExp-Enhancements-CLI-IDE-Git-AI-9d72cc68bcc44a6bafe95eab793235fb)

runleonarun committed:
Merge branch 'spring-launch-24' into spring-launch-release-notes

matthewshaver committed:
Update website/docs/docs/build/unit-tests.md

matthewshaver committed:
Merge branch 'spring-launch-24' into spring-launch-lifecycle-updates

matthewshaver committed:
spring launch lifecycle updates (#5466)

this PR removes any lifecycle updates/statuses based on the upcoming
spring launch.

mirnawong1 committed:
Merge branch 'current' into spring-launch-24

matthewshaver committed:
Update website/docs/docs/cloud/dbt-assist.md

matthewshaver committed:
Changing to expandables

matthewshaver committed:
Apply suggestions from code review

matthewshaver committed:
Update website/docs/docs/cloud/dbt-assist-data.md

mirnawong1 committed:
Update website/docs/docs/cloud/about-cloud/about-dbt-cloud.md

mirnawong1 committed:
Update dbt-assist.md

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-release-notes

mirnawong1 committed:
Merge branch 'current' into spring-launch-24

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-release-notes

Commit:
legal changes

Commit:
turn to headers

Commit:
h4

Commit:
remove this per legal

Commit:
legal updates

Commit:
updated

Commit:
fold brenda's feedback

Commit:
remove

Commit:
move to ide

mirnawong1 committed:
Update develop-in-the-cloud.md

Commit:
add to the ide

mirnawong1 committed:
Update develop-in-the-cloud.md

Commit:
add clarifying detail

Commit:
brenda feedback

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-dbt-assist

Commit:
fix conflict

Commit:
Merge branch 'spring-launch-dbt-assist' of https://github.com/dbt-labs/docs.getdbt.com into spring-launch-dbt-assist

Commit:
final tweaks

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-release-notes

mirnawong1 committed:
release note for spring launch (#5493)

this PR adds some release notes for the spring launch event.

this pr should be merged on may 14th at 12pm est to coincide with the
showcase launch event.

mirnawong1 committed:
Merge branch 'current' into spring-launch-24

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-dbt-assist

mirnawong1 committed:
Update develop-in-the-cloud.md

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes.md:
```diff
@@ -18,8 +18,65 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
+
+- **New:** Native support for Microsoft Fabric in dbt Cloud is now GA. This feature is powered by the [dbt-fabric](https://github.com/Microsoft/dbt-fabric) adapter. To learn more, refer to [Connect Microsoft Fabric](/docs/cloud/connect-data-platform/connect-microsoft-fabric) and [Microsoft Fabric DWH configurations](/reference/resource-configs/fabric-configs). There's also a [quickstart guide](ht ...
+
+- **New:** [dbt Mesh](ttps://docs.getdbt.com/best-practices/how-we-mesh/mesh-1-) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.co ...
```
```suggestion
- **New:** [dbt Mesh](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-1-) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).
```

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes.md

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes.md:
```diff
@@ -18,8 +18,65 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
+
+- **New:** The VS Code extension [Power user for dbt Core and dbt Cloud](https://docs.myaltimate.com/arch/beta/) is now available in beta for [dbt Cloud CLI](/docs/cloud/cloud-cli-installation#vs-code-extension) users. The extension accelerates dbt and SQL development and includes features such as generating models from your source definitions or SQL, updating dbt model using natural language, an ...
+
+- **New:** MetricFlow enables you to now add [metrics as dimensions](/docs/build/ref-metrics-in-filters) to your metric filters to create more complex metrics and gain more insights. Available for all users.
```
```suggestion
- **New:** MetricFlow enables you to now add metrics as dimensions to your metric filters to create more complex metrics and gain more insights. Available for all users.
```

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes.md

mirnawong1 committed:
Merge branch 'spring-launch-24' into spring-launch-dbt-assist

mirnawong1 committed:
adding new docs for dbt assist (#5478)

this pr adds new docs for dbt assist as reviewed/discussed with Greg and
team.

[notion
project](https://www.notion.so/dbtlabs/DevExp-Enhancements-CLI-IDE-Git-AI-9d72cc68bcc44a6bafe95eab793235fb)

mirnawong1 committed:
Update cloud-cli-installation.md

mirnawong1 committed:
Update release-notes.md

Commit:
update link

Commit:
remove s

runleonarun commented on website/docs/docs/dbt-versions/release-notes.md:
```diff
@@ -18,8 +18,65 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
+
+- **New:** Native support for Microsoft Fabric in dbt Cloud is now GA. This feature is powered by the [dbt-fabric](https://github.com/Microsoft/dbt-fabric) adapter. To learn more, refer to [Connect Microsoft Fabric](/docs/cloud/connect-data-platform/connect-microsoft-fabric) and [Microsoft Fabric DWH configurations](/reference/resource-configs/fabric-configs). There's also a [quickstart guide](ht ...
+
+- **New:** [dbt Mesh](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-1) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.co ...
```
Fixing to fix failing test
```suggestion
- **New:** dbt Mesh is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).
```

runleonarun committed:
Update website/docs/docs/dbt-versions/release-notes.md

nghi-ly committed:
GA of dbt Explorer (#5488)

## What are you changing in this pull request and why?

GA of dbt Explorer 

The [Docs
Plan](https://www.notion.so/dbtlabs/GA-Explorer-and-all-its-features-9683fd8a5d8e4da58955f89212effba0?pvs=4)
for this launch lists all the planned changes.

## Checklist
- [x] Review the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [x] For [docs
versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning),
review how to [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Needs review from PM and DX

---------

Co-authored-by: Mirna Wong <89008547+mirnawong1@users.noreply.github.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>
Co-authored-by: mirnawong1 <mirna.wong@fishtownanalytics.com>

runleonarun committed:
Merge branch 'current' into spring-launch-24

Commit:
add low code and move dbt assist

Commit:
sl, explorer, cloud cli updates

Commit:
remove link

Commit:
tweak language

Commit:
remove staging beta flags

Commit:
remove beta

Commit:
add staging

mirnawong1 committed:
Merge branch 'current' into spring-launch-24

Commit:
remove s

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes.md:
```diff
@@ -18,8 +18,69 @@ Release notes are grouped by month. For customers using dbt Virtual Private Clou
+
+- **New:** MetricFlow enables you to now add metrics as dimensions to your metric filters to create more complex metrics and gain more insights. Available for all dbt Semantic Layer users.
+
+- **New:** [Staging environment](/docs/deploy/deploy-environments#staging-environment) is now GA. Use staging environments to grant developers access to deployment workflows and tools while controlling access to production data. Available to dbt Cloud users.
```
```suggestion
- **New:** [Staging environment](/docs/deploy/deploy-environments#staging-environment) is now GA. Use staging environments to grant developers access to deployment workflows and tools while controlling access to production data. Available to all users.
```

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes.md

Commit:
add cloud users

matthewshaver committed:
Updating lifecycles

matthewshaver committed:
Merge branch 'spring-launch-24' into beta-removal

matthewshaver committed:
Updating lifecycles (#5505)

## What are you changing in this pull request and why?

Updating lifecycle statuses 

## Checklist
<!--
Uncomment when publishing docs for a prerelease version of dbt:
- [ ] Add versioning components, as described in [Versioning
Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
so my content adheres to these guidelines.
- [ ] For [docs
versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning),
review how to [version a whole
page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
and [version a block of
content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this
PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links
that point to deleted pages

Commit:
update model per sara

Commit:
update

mirnawong1 committed:
Merge branch 'current' into spring-launch-24

runleonarun reviewed (approved):
let's ship this puppy 🐶
